bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/SharedDataClientWrapper.java

```diff
@@ -110,7 +110,8 @@ public void close() throws IOException {
   }
 
   @Override
-  public void setCaching(int caching) {
+  public ResultScanner readRows(Query.QueryPaginator paginator, int requestedPageSize) {
     // unimplemented.
+    return null;
```

Comment:
can we add this method to ClientWrapper and call delegate.readRows(QueryPaginator) here?

Reply:
Done